src/typescript/azure-openapi-validator/rules/EnumMustNotHaveEmptyValue.ts

```diff
@@ -15,8 +16,8 @@ rules.push({
   appliesTo_JsonQuery: "$..*[?(@.enum)]",
   *run(doc, node, path) {
     const msg: string = `Enum value must not contain empty value.`
-    if (node.enum !== undefined) {
-      const enumList: string[] = node.enum
+    if (path.indexOf("x-ms-examples") === -1 && isValidEnum(node)) {
```

Comment:
Why check x-ms-examples

Reply:
it will cause false positive, no need to check model in x-ms-examples